tests/integrated/test-snb/test_snb.cxx

```diff
@@ -76,7 +76,7 @@ int main(int argc, char** argv) {
     auto Te = factory.create3D("5 + cos(y)");
     auto Ne = factory.create3D("1e18 * (1 + 0.5*sin(y))");
 
-    mesh->communicate(Te, Ne);
+    Ne.getMesh()->communicate(Te, Ne);
```

Comment:
We now use `getMesh()` a bunch of times in this `main` - maybe it's simpler to just use `using bout::globals::mesh`?